pkg/configuration/configuration.go

```diff
@@ -143,6 +143,12 @@ const (
 	// varVerificationCodeExpiresInMin is used to set the amount of time (in minutes) that a verification code is active for before expiring
 	varVerificationCodeExpiresInMin     = "verification.code_expires_in_min"
 	DefaultVerificationCodeExpiresInMin = 5
+
+	// varWoopraDomain contains the woopra domain
+	varWoopraDomain = "woopra.domain"
+
+	// varSegmentWriteKey contains the segment write key
+	varSegmentWriteKey = "segment.write.key"
```

Comment:
it makes sense to separate parent/child so i've updated here:  ab173028c923ed60f3fafc5730402651f97364e7

we do it for some twilio keys but not for all of them, we should update it at some point for twilio and mailgun so it's consistent. maybe i'll open a jira so that i remember to do that after this is done!